extensions/_states/heroku.py

```diff
@@ -0,0 +1,181 @@
+# -*- coding: utf-8 -*-
+'''
+State for managing heroku apps.
+
+.. versionadded:: 2015.5.0
```

Comment:
Should modify or omit the `versionadded` line since it's not in upstream salt and that's not the current version :simple_smile: